portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,19 +14,47 @@
 
 package com.google.sps.servlets;
 
+import com.google.gson.Gson;
+import com.google.sps.data.Comment;
+import com.google.sps.data.CommentsManager;
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.List;
+import java.util.ArrayList;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-
+  private static final String COMMENT_INPUT = "comment-input";
+  private static final CommentsManager COMMENTS_MANAGER = new CommentsManager();
```

Comment:
the commentManager, even though it can be final, shouldn't be final, as it is not a constant
since technically we are modifying it
lets rename it to just commentsManager

Reply:
Okay!